internal/service/alarms/serve.go

```diff
@@ -13,11 +13,12 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/alertmanager"
+
```

Comment:
let's put it together with the other imports

Reply:
I don't modify it manually actually. Simply run `golangci-lint run --fix --config .golangci.yml` and let it do its thing.

Reply:
weird, we've been grouping this kind of imports together. Lint locally and CI do not complain.

Reply:
updated manually